test/python/transpiler/test_unroller.py

```diff
@@ -272,18 +272,18 @@ def test_simple_unroll_parameterized_with_expressions(self):
         self.assertEqual(circuit_to_dag(expected), unrolled_dag)
 
     def test_definition_unroll_parameterized(self):
-        """Verify that unrolling complex gates with parameters raises."""
+        """Verify that unrolling complex gates with parameters does not raise."""
         qr = QuantumRegister(2)
         qc = QuantumCircuit(qr)
 
         theta = Parameter('theta')
 
-        qc.cu1(theta, qr[0], qr[1])
+        qc.cu1(theta * theta, qr[0], qr[1])
```

Comment:
why?

Reply:
To make sure we cover `ParameterExpression`s and not just `Parameter`s in f5171b5a, but it might make sense to have both.

Reply:
Returned single theta case in 7d07f3c6.

Reply:
I'm not fully sure how a why we need to distinguish between `Parameter` and `ParameterExpression`... but that's another dicussion...